### PR TITLE
Add makeshift blindfold made from natural materials

### LINKED ISSUE #48846: Add a cordage blindfold
# Is your feature request related to a problem? Please describe.
No logical reason you shouldn't be able to wrap a few strings of thick cordage around your head, covering your eyes, to help you sleep in the light.

# Describe the solution you'd like
Add a "cordage blindfold" that can be crafted from a few (five should probably be enough) long cordage pieces.

# Describe alternatives you've considered
Innawoods players having unnecessary difficulty making such a simple utility item that, from a realism POV, should be able to be crafted innawoods.

# Additional context
None.

### PR DESCRIPTION
**Pull request by LohTek**
#### Summary
Content "Adds makeshift blindfold made of natural materials"

#### Purpose of change
It will provide a way for survivors lost in the woods a means of making a blindfold to sleep when it is still bright outside 
Fixes #48846 Add a cordage blindfold.

#### Describe the solution
Provides a means of making a blindfold from natural materials easily found in the forest.

#### Describe alternatives you've considered
In reference to #48846 I find that wrapping cordage around ones face to be unrealistic as laying down with this upon your head will cause damage to the skin and make it uncomfortable to sleep; a better solution is to fashion a simple grass woven material to cover the eyes and allow one to sleep.

#### Testing
Created object, wore object in game it performs as expected; disassembly is impossible due to the crude nature of the object all materials would be destroyed in the process as the way in which they are attached is destructive to the components. (according to my decisions)

#### Additional context
Blindfold1

### CODE REVIEW

**Comment by LohTek** [comment]
Git desktop is REALLY BAD

File: data/json/items/tool_armor.json [hunk 1]
```diff
@@ -92,6 +92,27 @@
     "use_action": { "target": "tourniquet_upper", "msg": "You adjust the tourniquet.", "menu_text": "Adjust", "type": "transform" },
     "flags": [ "WATER_FRIENDLY", "OUTER", "TOURNIQUET", "OVERSIZE" ]
   },
+  {
+    "id": "makeshift blindfold",
```
Comment (LaVeyanFiend) [inline data/json/items/tool_armor.json:96]:
```suggestion
    "id": "makeshift_blindfold",
```

File: data/json/recipes/armor/head.json [hunk 2]
```diff
@@ -41,6 +41,20 @@
     "using": [ [ "sewing_standard", 20 ] ],
     "components": [ [ [ "felt_patch", 4 ] ] ]
   },
+  {
+    "result": "makeshift blindfold",
```
Comment (LaVeyanFiend) [inline data/json/recipes/armor/head.json:45]:
```suggestion
    "result": "makeshift_blindfold",
```

**Comment by Shodan14** [comment]
Do we really need a makeshift blindfold? Why not just have it make a normal blindfold?

**Comment by actual-nh** [comment]
> Do we really need a makeshift blindfold? Why not just have it make a normal blindfold?

Because it's unreasonable for you to be able to make a cotton blindfold out of grass and twine?

**Comment by Shodan14** [comment]
> 
> 
> > Do we really need a makeshift blindfold? Why not just have it make a normal blindfold?
> 
> Because it's unreasonable for you to be able to make a cotton blindfold out of grass and twine?
The description of the item could be made more generic to cover all kinds of blindfolds. I'd expect a makeshift item to have some relevant mechanical difference from the "real" one.

**Comment by LohTek** [comment]
> 
> 
> > > Do we really need a makeshift blindfold? Why not just have it make a normal blindfold?
> > 
> > 
> > Because it's unreasonable for you to be able to make a cotton blindfold out of grass and twine?
> 
> The description of the item could be made more generic to cover all kinds of blindfolds. I'd expect a makeshift item to have some relevant mechanical difference from the "real" one.

The purpose of it being makeshift was to make use of the fact that we can basket weave in the game, its pretty dang hard to make cloth in the middle of no where, so normal weaving with a loom is out. We already have a regular blindfold made with many different cloth materials, this one is for a specific style of play to make it possible to sleep.

File: data/json/items/tool_armor.json [hunk 3]
```diff
@@ -92,6 +92,27 @@
...
+    "material": [ "dry_plant" ],
+    "symbol": "[",
+    "looks_like": "scarf",
+    "color": "yellow",
+    "covers": [ "eyes" ],
+    "coverage": 95,
+    "encumbrance": 10,
+    "warmth": 0,
+    "material_thickness": 0.1,
+    "environmental_protection": 1,
```
Comment (actual-nh) [inline data/json/items/tool_armor.json:113]:
```suggestion
    "environmental_protection": 0,
```

File: data/json/recipes/armor/head.json [hunk 4]
```diff
@@ -41,6 +41,20 @@
...
+    "result": "makeshift_blindfold",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_HEAD",
+    "skill_used": "survival",
+    "difficulty": 2,
+    "skills_required": [ [ "tailor", 1 ] ],
+    "time": "20 m",
+    "autolearn": true,
```
Comment (Maleclypse) [inline data/json/recipes/armor/head.json:54]:
```suggestion
    "autolearn": true,
    "reversible": true,
```

**Review by Maleclypse — APPROVED** [review]
Made changes to in order to approve for easy merge.  Approved.

**Pull request merged**